src/modules/dashboard/components/Dialogs/ControlSafeDialog/validation.ts

```diff
@@ -0,0 +1,255 @@
+import { isHexString } from 'ethers/utils';
+import { toFinite } from 'lodash';
+import { defineMessages, MessageDescriptor } from 'react-intl';
+import * as yup from 'yup';
+
+import { intl } from '~utils/intl';
+import { validateType } from '~utils/safes';
+import { TransactionTypes } from './constants';
+
+const MSG = defineMessages({
+  requiredFieldError: {
+    id: 'dashboard.ControlSafeDialog.requiredFieldError',
```

Comment:
You are missing the validation part in all the ids, i.e: 
id: 'dashboard.ControlSafeDialog.validation.requiredFieldError'

Reply:
@Julianahlk Thanks. Please re-review when you can.